coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1585,6 +1615,42 @@ func (s *server) notifyWorkspaceDeleted(ctx context.Context, workspace database.
 	}
 }
 
+func (s *server) notifyTemplateOwnerAboutManualBuildFailure(ctx context.Context, workspace database.Workspace, build database.WorkspaceBuild) {
```

Comment:
```suggestion
func (s *server) notifyTemplateManualBuildFailure(ctx context.Context, workspace database.Workspace, build database.WorkspaceBuild) {
```

Reply:
What's going on with this function? 🤔

Reply:
This is a good question. I think I may have created it for reference and forgot to remove it.